lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -485,7 +497,6 @@ function setupEventListeners(transport) {
     const twilioConnection = createOrResetTwilioConnection();
     twilioConnection.once('close', disconnect);
     twilioConnection.on('message', handleMessage);
-    twilioConnection.once('open', connect);
```

Comment:
but I suggest we move  following lines to `createOrResetTwilioConnection` so that a) we can also unhook `close` from old connection. and b) It would look more obvious if hooking and unhooking was in the same function.  
```
    twilioConnection.once('close', disconnect);
    twilioConnection.on('message', handleMessage);
```